service/controller/v24/resource/cpi/create.go

```diff
@@ -5,12 +5,12 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 	"github.com/giantswarm/microerror"
 
-	"github.com/giantswarm/aws-operator/service/controller/v24/adapter"
 	"github.com/giantswarm/aws-operator/service/controller/v24/controllercontext"
 	"github.com/giantswarm/aws-operator/service/controller/v24/key"
-	"github.com/giantswarm/aws-operator/service/controller/v24/templates"
+	cpitemplate "github.com/giantswarm/aws-operator/service/controller/v24/templates/cloudformation/cpi"
```

Comment:
I am not super happy with that. I would like to not deal with an alias here. Is there an alternative for the package? Should the package just be `cpitemplate`? Should we maybe move the template and data structure into the resource? That would make sense to me. We could even have a `template` folder within the resource so that would be clear.

Reply:
I would store template in the same package. Don't see problem with that. Especially when at some point we have generic CF resource.

If not why not just use `cpi`?

Reply:
> If not why not just use cpi?

I think I know: `var cpi *cpi.CPI`.